hircluster.c

```diff
@@ -1848,6 +1891,27 @@ int redisClusterSetOptionConnectNonBlock(redisClusterContext *cc) {
     return REDIS_OK;
 }
 
+/**
+ * Configure a password used when connecting to password-protected
+ * Redis instances. (See Redis AUTH command)
+ */
+int redisClusterSetOptionPassword(redisClusterContext *cc,
+                                  const char *password) {
+
+    if (cc == NULL || password == NULL) {
+        return REDIS_ERR;
+    }
+
+    if (strlen(password) > CONFIG_AUTHPASS_MAX_LEN) {
```

Comment:
Just a question: Would be equivalent? `if (strlen(password) >= sizeof(cc->password))`. If yes, then I guess it's a matter of taste...

Reply:
ah, yes, would be the same